apps/lint_exemptions.js

```diff
@@ -1251,12 +1251,6 @@ module.exports = {
       "no-undef"
     ]
   },
-  "apps/android/boot.js": {
```

Comment:
is this supposed to be removed, or should we just update the hash?

Reply:
Let's drop it - I've sorted the lint warning in 350cb9d8126029536a471496a9e63b6681ba12c7